<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutDialog</class>
 <widget class="QDialog" name="ShortcutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add to Steam</string>
  </property>
  <widget class="QGroupBox" name="shortcut_settings_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>530</width>
     <height>216</height>
    </rect>
   </property>
   <property name="title">
    <string>Shortcut Settings</string>
   </property>
   <widget class="QWidget" name="shortcut_settings_form">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>530</width>
      <height>196</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="verticalSpacing">
      <number>-1</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="allow_external_access_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Access Externally?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="allow_external_access_checkbox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="external_dns_label">
       <property name="text">
        <string>External IP Address or DNS Entry</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="external_dns_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="local_ssid_label">
       <property name="text">
        <string>What's the SSID of your home wifi?</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="local_ssid_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="mode_combo_box"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mode_combo_box_label">
       <property name="text">
        <string>Screen Mode</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="passcode_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="passcode_label">
       <property name="text">
        <string>Pincode (Optional)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="choose_artwork_button">
       <property name="text">
        <string>Choose Artwork</string>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="choose_artwork_label">
       <property name="text">
        <string>Artwork</string>
       </property>
       <property name="visible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="add_to_steam_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>216</y>
     <width>510</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to Steam</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
